great_expectations_cloud/agent/agent.py

```diff
@@ -126,6 +127,15 @@ def __init__(self: Self):
             # suppress warnings about GX version
             warnings.filterwarnings("ignore", message="You are using great_expectations version")
             self._context: CloudDataContext = get_context(cloud_mode=True)
+            if not enable_progress_bars:
```

Comment:
This LGTM - just a heads up that this needs to be done in get_data_context() in the runner (since the context is created for each run).